lsp/server/package.json

```diff
@@ -23,6 +24,10 @@
         "@babel/preset-env": "^7.25.7",
         "@types/babel__traverse": "7.20.6",
         "@types/babel-core": "6.25.10",
-        "graphql": "^16.9.0"
+        "@types/mocha": "^10.0.7",
+        "@types/sinon": "^17.0.3",
```

Comment:
child project could reference root project's modules. So `@type/mocha` and `@types/sinon` is not necessary here.